Neos.ContentRepository.Export/src/Event/ValueObject/ExportedEvent.php

```diff
@@ -24,10 +24,13 @@ public function __construct(
 
     public static function fromRawEvent(Event $event): self
     {
+        $payload = \json_decode($event->data->value, true, 512, JSON_THROW_ON_ERROR);
+        // unset content stream id as this is overwritten during import
+        unset($payload['contentStreamId']);
```

Comment:
IMO we can/should remove the workspace name, too:
```suggestion
        unset($payload['contentStreamId'], $payload['workspaceName']);
```

Reply:
adjusted